spec/factories/partners/user.rb

```diff
@@ -1,13 +0,0 @@
-FactoryBot.define do
```

Comment:
Oh I see, is it because we want the user to be associated with a partner in the diaper db right? I think we have a `partner` model in both diaper db (`::Partner`) and partner db (`Partners::Partner`) at the moment, right? Is the eventual plan to merge the DBs into one or do you think we'll keep them separate?